compass/sign/RemoteSignatureSource.java

```diff
@@ -56,11 +57,28 @@ public RemoteSignatureSource(String uri,
    * @param uri the URI of the host to connect to
    */
   public RemoteSignatureSource(String uri) {
-    this(ManagedChannelBuilder.forTarget(uri).usePlaintext().idleTimeout(5, TimeUnit.SECONDS).build());
+    this.channelBuilder = getManagedChannelBuilder(uri);
+    this.serviceStub = SignatureSourceGrpc.newBlockingStub(channelBuilder.build());
+  }
+
+  private ManagedChannelBuilder getManagedChannelBuilder(String uri,
```

Comment:
Maybe call it `createManagedChannelBuilder`

It helps me understand the code better